### DISCUSSION TITLE
pg_stat_statements extension support

### DISCUSSION THREAD

**Discussion by lfittl** [neondatabase/neon#1948] [Ideas]
First of all, congrats on the launch!

We'd love to provide a direct [pganalyze](https://pganalyze.com/) integration with the hosted Neon service - and one of the extensions we rely on is pg_stat_statements (for collecting query statistics), which I'm sure other folks would also find useful.

I know its not there yet today in the beta, but figured we can have a tracking item here in case its added in the future. If you do add it, it would also be great if you include support for setting the pg_monitor role on a user, so one can have a dedicated monitoring user configured on the database.

Wish y'all all the best :)

**Comment by danieltprice** [comment]
pg_stat_statements is now supported. Please see:
https://neon.tech/docs/extensions/pg-extensions

**Comment by ghost** [comment]
@lfittl Do you guys support neon now?

**Comment by analytically** [comment]
@lfittl any update?

**Comment by rchasman** [comment]
Please our team is desperate to purchase pganalyze if we could just connect it to Neon! 
<img width="1035" height="527" alt="Screen Shot 2026-02-26 at 5 05 18 PM" src="https://github.com/user-attachments/assets/0c9c8fdb-1150-4fd7-9532-8ae7296b4ac5" />
